google/cloud/firestore_v1/base_collection.py

```diff
@@ -550,23 +550,34 @@ def find_nearest(
         query_vector: Vector,
         limit: int,
         distance_measure: DistanceMeasure,
+        distance_result_field: Optional[str] = None,
+        distance_threshold: Optional[Union[int, float]] = None,
```

Comment:
Maybe we should add a `*` before these optional args to make them keyword-only? That buys us a bit of wiggle-room to re-order them later, if we end up adding more args in the future

(I probably would have preferred to make everything after query_vector keyword-only, but it's too late for that change)

Reply:
Done.